db/migrate/20240924095435_back_fill_claims_started_at.rb

```diff
@@ -0,0 +1,24 @@
+class BackFillClaimsStartedAt < ActiveRecord::Migration[7.0]
+  def up
+    execute <<-SQL
+      UPDATE claims
+      SET started_at = journeys_sessions.created_at
+      FROM journeys_sessions
+      WHERE claims.journeys_session_id = journeys_sessions.id
+      AND claims.started_at IS NULL
+    SQL
+
+    execute <<-SQL
+      UPDATE claims
+      SET started_at = created_at
+      WHERE journeys_session_id IS NULL
+      AND claims.started_at IS NULL
+    SQL
+
+    change_column_null :claims, :started_at, false
```

Comment:
feels risky without a prior run? might save you more time if we do this change after confirmation? or happy to keep in if you're feeling confident, i don't think there's any material risk

Reply:
We're setting this on new claims in the previous PR and we're using the created_at timestamp which should be present on all records as rails is setting it. But yeah I think for avoiding risk we'll split this out into a separate pr (and also because I forgot to remove the not null constraint in the down migration 🙈)

Reply:
Have moved the migration to make the column non nullable to this pr https://github.com/DFE-Digital/claim-additional-payments-for-teaching/pull/3224/files